common/core/src/errors.ts

```diff
@@ -370,3 +370,18 @@ export class DeviceTimeoutError extends Error {
     Error.captureStackTrace(this, this.constructor);
   }
 }
+
+/**
+ * Error thrown when an operation is cancelled
```

Comment:
>operation [](start = 24, length = 9)

is it generic or a specific type of DPS operation?

Reply:
generic.  updated comments to reflect this.

---
In reply to: [141476392](https://github.com/Azure/azure-iot-sdk-node/pull/93#discussion_r141476392) [](ancestors = 141476392)